source/appModules/taskmgr.py

```diff
@@ -0,0 +1,28 @@
+#appModules/taskmgr.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2018 NV Access Limited, Derek Riemer
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+import appModuleHandler
+from NVDAObjects import NVDAObject
+from NVDAObjects.UIA import UIA
+
+def appropriateDescendant(obj):
```

Comment:
Either a docstring here or a name that better explains what this function is for.

Reply:
I'd say the doc string is now clear, but still personally I'd prefer a name like `isChildOfRow`